webpack.common.js

```diff
@@ -0,0 +1,178 @@
+const path = require("path");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+const { CleanWebpackPlugin } = require("clean-webpack-plugin");
+const ForkTsCheckerWebpackPlugin = require("fork-ts-checker-webpack-plugin");
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+const CopyPlugin = require("copy-webpack-plugin");
+const ESLintPlugin = require("eslint-webpack-plugin");
+const ProvidePlugin = require("webpack").ProvidePlugin;
+
+module.exports = {
+  entry: "./src/index.tsx",
+  mode: "development",
+  devtool: "source-map",
+  optimization: {
+    usedExports: true,
+    splitChunks: {
+      cacheGroups: {
+        main: {
+          test: /[\\/]src[\\/]/,
+          name: "main",
+          chunks: "all",
+        },
+        "vendor-react-redux": {
+          test: /[\\/]node_modules[\\/](react|react-dom|react-router|react-router-dom|react-redux|redux|redux-thunk|@reduxjs)[\\/]/,
+          name: "vendor-react-redux",
+          chunks: "all",
+        },
+        "vendor-stellar": {
+          test: /[\\/]node_modules[\\/](@stellar|stellar)[\\/]/,
+          name: "vendor-stellar",
+          chunks: "all",
+        },
+        "vendor-wallets": {
+          test: /[\\/]node_modules[\\/](@albedo|@ledgerhq|trezor)[\\/]/,
+          name: "vendor-wallets",
+          chunks: "all",
+        },
+        "vendor-other": {
```

Comment:
So... I was getting a very large chunk `main` without it, but I figured out why. :sweat_smile: It looks like by default the file name is `main`, so it was adding all the `src` stuff to it as well. I will rename `src` chunk `source` and remove `vendor-other`. This will create `main` for everything else. Thanks for questioning this stuff, @piyalbasu ! This is a cleaner solution. :+1: